### PR TITLE
Studio bash logger

### PR DESCRIPTION
**Pull request by DavidGOrtega**
Adds bash logger to studio as #733 and #734 😁 

 - closes partially https://github.com/iterative/leo-server/issues/36

### CODE REVIEW

**Comment by casperdcl** [comment]
10x smaller than #733; I like :P

**Review by 0x2b3bfa0 — APPROVED** [review]
Static review

File: task/common/machine/machine-script.sh.tpl [hunk 1]
```diff
@@ -13,16 +13,51 @@ sleep 20; while pgrep rclone > /dev/null; do sleep 1; done
 source /opt/task/credentials
 (systemctl is-system-running | grep stopping) || leo stop --cloud="$TPI_TASK_CLOUD_PROVIDER" --region="$TPI_TASK_CLOUD_REGION" "$TPI_TASK_IDENTIFIER";
 END
-
 chmod u=rwx,g=rx,o=rx /usr/bin/tpi-task-shutdown
 
+sudo tee /usr/bin/tpi-task-studio-log << 'END'
```
Comment (tasdomas) [inline task/common/machine/machine-script.sh.tpl:18]: Maybe at least add a few comment explaining what this is for.

It's not like `machine-script.sh.tpl` is  a shining example of literate programming.

**Comment by tasdomas** [comment]
> 10x smaller than #733; I like :P
2.5 times, bearing in mind that #733 had tests

File: task/common/machine/machine-script.sh.tpl (hunk 1 above)
Comment (DavidGOrtega) [reply task/common/machine/machine-script.sh.tpl:18]: I think the name suffices `tpi-task-studio-log` however it should be probably `tpi-task-studio-logger`

**Comment by DavidGOrtega** [comment]
> > 10x smaller than #733; I like :P
> 
> 2.5 times, bearing in mind that #733 had tests

Indeed your work is going to be used in leo-server at least

**Comment by omesser** [comment]
@0x2b3bfa0 @dacbd  - Ping for status. Since it was approved, can be merged  & released ?

**Comment by 0x2b3bfa0** [comment]
Probably 😅

**Comment by dacbd** [comment]
getting there, please hold 🙃

**Comment by DavidGOrtega** [comment]
@dacbd Its ready to be merged. It worked perfectly. We are still missing the first provisioning state but that should be done in  tpi

**Pull request merged**